content/app-dev/create-actor/update.md

```diff
@@ -71,3 +84,5 @@ _This is the way production software services were meant to be_.
 
 No-fuss version management. Secure software supply-chain.
 Zero-downtime upgrades.
+
+[^1]: Because we're running the wasmCloud dashboard in a browser, the `Upload File` form is prevented from knowing exactly where files are on disk for security reasons. However, since the actual Elixir application has access to files on disk, an absolute path (or relative path from where you started the host) is accessible for hot reloading.
```

Comment:
replace "since the Elixir application has" with "since the host process, an Elixir application, has"